frontend/src/component/EditIndexCard.tsx

```diff
@@ -11,24 +13,29 @@ export default function EditIndexCard({addNewIndexCard}: EditIndexCardProps) {
     const [term1, setTerm1] = useState("")
     const [term2, setTerm2] = useState("")
 
-    function updateTerm(option: number, term: string) {
+    function updateTerm(option: 1 | 2, term: string) {
```

Comment:
Ich bevorzuge hier diese schreibweiße, ist aber geschmackssache...
```suggestion
    const updateTerm = (option: 1 | 2, term: string) => {
```

Reply:
Hmm, habe das Gefühl egal wie man es macht gibts ein Chaos :D muss ich mir mal überlegen